src/ctl.c

```diff
@@ -68,6 +69,9 @@ CTL_PROTO(max_background_threads)
 CTL_PROTO(thread_tcache_enabled)
 CTL_PROTO(thread_tcache_max)
 CTL_PROTO(thread_tcache_flush)
+CTL_PROTO(thread_tcache_sizes_max_cached_set)
+INDEX_PROTO(thread_tcache_size_i)
+CTL_PROTO(thread_tcache_size_i_max_cached_get)
```

Comment:
We don't usually mix `size` and `i` together, since the latter is often considered as index. Also we can't use the INDEX_PROTO this way, since it's limited to MIB_UNSIGNED which isn't enough for size_t.

Maybe do `thread.tcache.ncached_max.read_sizeclass()` (use the write input as the lookup size) and `thread.tcache.ncached_max.write()`? We will want to emphasize on the `ncached` to say it's not max bytes.

Reply:
Updated the name.